src/Umbraco.Core/Composing/Current.cs

```diff
@@ -208,6 +209,8 @@ public static IPublishedValueFallback PublishedValueFallback
         public static IVariationContextAccessor VariationContextAccessor
             => Factory.GetInstance<IVariationContextAccessor>();
 
+        public static IImageUrlGenerator ImageUrlGenerator
```

Comment:
Is any reason why we are using that in current instead of direct injecting in places of use?

Reply:
I could use Current.Factory instead, but all the existing extension methods need some way to get the current generator without already having an IImageUrlGenerator instance.